components/Cards/index.js

```diff
@@ -17,3 +17,68 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+function Card( heading, pic, authName ) {
+  // add elements
+  const card = document.createElement('div');
+  const headline = document.createElement('div');
+  const author = document.createElement('div');
+  const imgContainer = document.createElement('div');
+  const img = document.createElement('img');
+  const name = document.createElement('span');
+
+  // add classes
+  card.classList.add('card');
+  headline.classList.add('headline');
+  author.classList.add('author');
+  imgContainer.classList.add('img-container');
+  
+  // add content: 
+  headline.textContent = heading;
+  img.src = pic;
+  name.textContent = `By ${authName}`;
+
+  // append elements
+  card.appendChild(headline);
+  headline.appendChild(author);
+  author.appendChild(imgContainer);
+  imgContainer.appendChild(img);
+  author.appendChild(name);
+
+  return card;
+}
+
+let articleArray = [];
+const cardsContainer = document.querySelector('.cards-container');
+console.log(cardsContainer)
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+.then( response => {
+  // console.log(response.data.articles);
+  let articleList = response.data.articles;
+  Array.from(articleList);
```

Comment:
I looked this over for the Array.from(articleList). I do not think its needed here. You can comment it out and check the page. Should work fine. Otherwise great job.